src/main/solution_space/data_classes.py

```diff
@@ -30,34 +31,51 @@ def is_empty(self) -> bool:
 
 
 class Profile:
-    def __init__(self, age=consts.DEFAULT_VALUES.AGE.value, experience=consts.DEFAULT_VALUES.EXPERIENCE.value):
+    def __init__(self, age=consts.DEFAULT_VALUES.AGE.value, experience: Union[EXPERIENCE, DEFAULT_VALUES] = DEFAULT_VALUES.EXPERIENCE):
```

Comment:
Use type annotation in all cases or don't use in all cases

Reply:
I'll do it later when we find out what to do with enums :)